File: src/main/java/cardgame/Player.java
package cardgame;

public class Player {
	private final String name;
	private char[] cards;
	
	public Player(String name) {
		this.name = name;
	}

	public char[] getCards() {
		return cards;
	}

	public void setCards(char[] cards) {
		this.cards = cards;
	}

	public String getName() {
		return name;
	}
}

File: player.js
module.exports = class Player {

    constructor(name) {
        this.name = name;
        this.cards = [];
    }

    getCards() {
        return this.cards;
    }

    setCards(cards) {
        this.cards = cards;
    }

    getName() {
        return this.name;
    }
}
